### DISCUSSION TITLE
"Overwhelmed by notifications? We've found some repositories..." Please stop

### DISCUSSION THREAD

**Discussion by mcclure** [community/community#39688] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

> Overwhelmed by notifications? We've found some repositories that may be causing notifications you don't need. [Update watching settings](https://github.com/watching)

Every couple months I get this message in my Notifications tab. It is never, ever useful. The "repositories that cause notifications I don't need" it guesses are always repositories I am watching on purpose. It is not helping to reduce the noise in my Notifications tab. It *is* the noise in my notifications tab. Nothing I do makes it go away. I close the "Overwhelmed?" box and it comes back awhile later.

I think the reason it keeps popping this up is I have a lot of notifications. I never remove notifications from my "Inbox" because at some point instead of the old, simple and obvious implementation (already-viewed notifications being automatically bumped to a separate page), it switched to this complex featureset with various checkboxes and buttons and three different boxes notifications can be sorted between and I just don't care. It is easier to just ignore the notification-sorting features and leave already-viewed notifications sitting in the inbox. This is fine, I have accepted it. But then *github* harasses me to clean up my notifications because it thinks the notifications tab is too messy. Leave me alone mom it's *my* room

(This box is asking me to check if I've done a search for similar discussions. Well, there [are similar discussions](https://github.com/community/community/discussions/5267). A lot of people seem annoyed by this box. But Github keeps showing it after I've dismissed it multiple times so it seems like they need one more discussion than they already had)

**Comment by ossanna16** [comment]
Hi there @mcclure and welcome to our community! Thank you for the feedback!

**Comment by justinmchase** [comment]
This 100%.

**Comment by adisib** [comment]
I am perfectly capable of managing what notifications I am interested in. What I am overwhelmed by are all these "Overwhelmed by notifications?" notifications. Please give us a way to turn it off. It has been incredibly annoying for a long time.

**Comment by dfalling** [comment]
I hate this thing too. It's not even accurate. It encourages me to unsubscribe from issues and repositories that I'm clearly interacting with. I'm never dismissing the notifications it claims I don't need. I'm always opening them, merging a branch, reassigning an issue...

**Comment by siph** [comment]
You can block this element with [`ublock`](https://github.com/gorhill/uBlock):
```
##.js-unwatch-suggestions
```

**Comment by srittau** [comment]
I also find it quite condescending. "Are you too stupid to manage your notifications or understand our UI?" I've been developing web apps for more than 20 years, and I've been using GitHub for who knows how long. I'm sure I'll be able to find the notification settings if I'm inclined to do so.

**Comment by awxiaoxian2020** [comment]
+1

**Comment by omerbensaadon** [comment, marked as answer]
We unshipped this :)
